nucypher/cli/actions/confirm.py

```diff
@@ -32,45 +35,77 @@
         expected_chain_name = 'DEPLOY'
     else:
         expected_chain_name = deployer_interface.client.chain_name
-    if click.prompt(f"Type '{expected_chain_name}' to continue") != expected_chain_name:
+    if click.prompt(f"Type '{expected_chain_name.upper()}' to continue") != expected_chain_name.upper():
         emitter.echo(ABORT_DEPLOYMENT, color='red', bold=True)
         raise click.Abort()
     return True
 
 
-def confirm_enable_restaking_lock(emitter, staking_address: str, release_period: int) -> bool:
+def confirm_enable_restaking_lock(emitter: StdoutEmitter, staking_address: str, release_period: int) -> bool:
+    """Interactively confirm enabling of the staking lock with user agreements."""
     emitter.message(RESTAKING_LOCK_AGREEMENT.format(staking_address=staking_address, release_period=release_period))
     click.confirm(CONFIRM_RESTAKING_LOCK.format(staking_address=staking_address, release_period=release_period), abort=True)
     return True
 
 
-def confirm_enable_restaking(emitter, staking_address: str) -> bool:
+def confirm_enable_restaking(emitter: StdoutEmitter, staking_address: str) -> bool:
+    """Interactively confirm enabling of the restaking with user agreements."""
     emitter.message(RESTAKING_AGREEMENT.format(staking_address=staking_address))
     click.confirm(CONFIRM_ENABLE_RESTAKING.format(staking_address=staking_address), abort=True)
     return True
 
 
-def confirm_enable_winding_down(emitter, staking_address: str) -> bool:
+def confirm_enable_winding_down(emitter: StdoutEmitter, staking_address: str) -> bool:
+    """Interactively confirm enabling of winding down with user agreements."""
     emitter.message(WINDING_DOWN_AGREEMENT)
     click.confirm(CONFIRM_ENABLE_WINDING_DOWN.format(staking_address=staking_address), abort=True)
     return True
 
 
-def confirm_staged_stake(staker_address, value, lock_periods) -> None:
-    click.confirm(CONFIRM_STAGED_STAKE.format(value=value,
+def confirm_staged_stake(staker_address: str, value: NU, lock_periods: int) -> bool:
+    """Interactively confirm a new stake reviewing all staged stake details."""
+    click.confirm(CONFIRM_STAGED_STAKE.format(nunits=str(value.to_nunits()),
+                                              tokens=str(value.to_tokens()),
                                               staker_address=staker_address,
                                               lock_periods=lock_periods), abort=True)
+    return True
 
 
-def confirm_large_stake(value: NU = None, lock_periods: int = None) -> None:
+def confirm_large_stake(value: NU = None, lock_periods: int = None) -> bool:
+    """Interactively confirm a large stake and/or a long stake duration."""
     if value and (value > NU.from_tokens(150000)):
         click.confirm(CONFIRM_LARGE_STAKE_VALUE.format(value=value), abort=True)
     if lock_periods and (lock_periods > 365):
         click.confirm(CONFIRM_LARGE_STAKE_DURATION.format(lock_periods=lock_periods), abort=True)
+    return True
```

Comment:
Was it necessary to make this function return a `bool`? Its returned value is never used, specially since the call to `click.confirm()` uses `abort=True`. Same comment to all the confirmation functions in here.

Reply:
Two motivations here: Personally, I like when a confirmation function returns `True`, secondly this allows `assert confirm_function()` in-test, which is a strong indication of function completion.